introduction_to_amazon_algorithms/jumpstart_text_to_image/Amazon_JumpStart_Text_To_Image.ipynb

```diff
@@ -332,7 +332,8 @@
     "def parse_response(query_response):\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #25.        return response_dict["generated_image"], response_dict["prompt"]
</pre><p>Nit: Can you simply do query_response["generated_image"], query_response["prompt"]?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/4598/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>